ocs/utils.py

```diff
@@ -606,3 +606,37 @@ def create_oc_resource(
         f.write(template)
     log.info(f"Creating rook resource from {template_name}")
     run_cmd(f"oc create -f {cfg_file}")
```

Comment:
In case the next comment is working for you, can you change this as well to:
```python
OCP.create(cfg_file)
```

Reply:
Done